.github/actions/smoke-test/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o errexit 
+set -o nounset 
+
+readonly TEMPLATE_ID="$1" 
+readonly SRC_DIR="/tmp/${TEMPLATE_ID}"
+
+echo "Running Smoke Test"
+
+readonly ID_LABEL="test-container=${TEMPLATE_ID}"
+devcontainer exec --workspace-folder "${SRC_DIR}" --id-label ${ID_LABEL} /bin/sh -c 'set -e && if [[ -f "test-project/test.sh" ]]; then cd test-project && if [[ "$(id -u)" = "0" ]]; then chmod +x test.sh; else sudo chmod +x test.sh; fi && ./test.sh; else ls -a; fi'
```

Comment:
This long line of code is challenging for a human to parse. Could it be reformatted a bit. Something like:

```
devcontainer exec \
  --workspace-folder "${SRC_DIR}" \
  --id-label ${ID_LABEL} \
  /bin/sh -c '\
    set -o errexit && \
    if [[ -f "test-project/test.sh" ]]; then \
      cd test-project && \
      if [[ "$(id -u)" = "0" ]]; then \
        chmod +x test.sh; \
     else \
       sudo chmod +x test.sh; \
     fi && \
    ./test.sh; \
  else \
    ls -a; \
  fi'
```